### ISSUE TITLE
Help content for internship/opportunity status

### ISSUE THREAD

**Issue by kbwagner** [openopps/OpenOppsTasks#159]
Create help page to display internship/opportunity status information
- should this be one help page for both internships and regular opportunities?  Or two separate?

**1) Internship Status pills**

Internship status values:
- Draft - The internship has been saved as a draft but not submitted
- Submitted - The internship has been submitted for approval to the community manager
- Approved - The internship has been approved by the community manager
- Open - the internship has been approved, the application start date for the selected cycle has passed, and the job has not yet been completed
- Completed - When the automated closeout process runs in ticket 2915, the internship opportunities for the entire cycle will be set to "Completed"

Where will the help display for student?
1) Student - Administrator view of  internship status:
![image](https://user-images.githubusercontent.com/18709918/59458672-f2fb5f00-8de8-11e9-982e-4bf02a8d5443.png)

**2) Federal opportunities**

Opportunity Status Values:
- Submitted 
- Draft
- Open
- Not Open
- In progress
- Completed
- Canceled

Where will these display?
1) Regular Opportunity landing page:
![image](https://user-images.githubusercontent.com/18709918/59458564-b7609500-8de8-11e9-994b-946ad7e0e136.png)

2) Regular opportunity admin view of status
![image](https://user-images.githubusercontent.com/18709918/59458884-7026d400-8de9-11e9-8b5b-f0273825ebf5.png)

Note: 
Open Opps currently uses "? Help" for links.

**Comment by kbwagner** [comment]
@dalenard21 - any updates on this one?  about to add one for applicants too.

**Comment by kbwagner** [comment]
This is platform ticket 3615

**Comment by dalenard21** [comment]
@kbwagner here is my first draft of the help content for the opportunity status.  

We do have definitions in the help center for 'open', 'in progress' and 'completed' for the filters.  So, I kept the definitions similar (but not exactly the same, since this help page is more for the creators).  Let me know what you think.

[Open-opps-help-center-what-does-the-opportunity-status-mean_20190620.docx](https://github.com/openopps/OpenOppsTasks/files/3315093/Open-opps-help-center-what-does-the-opportunity-status-mean_20190620.docx)

**Comment by kbwagner** [comment]
@dalenard21 - this looks good thanks!
- Same question - do we have a URL?  Or does Macon do that now?

**Comment by dalenard21** [comment]
@kbwagner @Piglady64 

Use this URL for production:
https://usajobs.github.io/openopps-help/opportunity/opportunity-status/

Here is the page in the test 'federalist' site:
https://federalist-proxy.app.cloud.gov/site/usajobs/openopps-help/opportunity/opportunity-status/

Let me know if there are other changes.

Once the release is ready to go to production, I guess Brice will have to push this change out with a GitHub build?  We can discuss closer to the release date.

**Comment by kbwagner** [comment]
@dalenard21 thanks...I updated the ticket.

Yes Brice says he will do it but we'll need to remind him since it's a new deployment task.

**Comment by dalenard21** [comment]
@kbwagner this page is now in production.  I'm closing this issue, but if any of the definitions need to change, just submit a new issue.

https://usajobs.github.io/openopps-help/opportunity/opportunity-status/
